roles/tsuru_api/tasks/main.yml

```diff
@@ -28,6 +28,10 @@
     tsuru-admin target-remove local;
     tsuru-admin target-add local http://127.0.0.1:{{api_port}} -s
 
+# Wait until the tsuru-server-api is up
```

Comment:
Can we omit this comment? I think the `name` and `wait_for` are descriptive enough by themselves, and there's a risk that in future the code may change in a way that contradicts the comment. A humorous take on this topic:

- https://twitter.com/nzkoz/status/538892801941848064
- https://twitter.com/nixcraft/status/574213958871085056